docs/example-opslevel-k8s.yaml

```diff
@@ -0,0 +1,20 @@
+version: 1.2.0
```

Comment:
does this match the output of `kubectl opslevel config sample --simple` ?  If so could we recommend doing just that so we don't have to maintain this file if things about the config change?

Reply:
No, this is an even simpler config I made that's about 10 lines shorter and just needs pods and not deploys so it's slightly easier to setup a test env for IMO. 

I will add a line telling them to use that command for another example of a config, but if we do change the config the output would change anyways and we'd have to re-write this file. 🙃